docs/rpc-ceph-install.md

```diff
@@ -0,0 +1,314 @@
+# Installation Procedures for Ceph in an RPC Customer Environment using rpc-ceph
+
+
+
+This process assumes that all servers already have either Ubuntu installed, all network configuration (incl bonding) is complete, and the servers are accessible via ssh.
+The servers involved in the deployment must be identified and the role(s) of each server clearly defined.  Every deployment needs "mons", "mgrs" and "osds".  Some deployments may also need RadosGW "rgws".  In a small deployment, a server may have all of these roles assigned.  In larger deployments, each server will typically be assigned fewer, or just a single role.
+In most cases, it is preferable to have separate networks for "Storage Front-end" (client access to the storage) and "Storage Back-end" (replication traffic within the cluster), but the "Storage Back-end" network is not required.
+The host from which the deployment will be managed must be identified.  This MAY be an "admin" server, "director" server or simply the first of the ceph storage servers, but it is USUALLY the first ceph storage server.  ALL of the following steps are to be performed on this deployment host.
+
+## Base Installation of Ceph using rpc-ceph
+
+### Create entries in /etc/hosts for all servers, specifying the IP address to which we can reach the servers via ssh:
+/etc/hosts
+Example **/etc/hosts** snippet:
+```bash
+192.168.0.101 ceph-stor01
+192.168.0.102 ceph-stor02
+192.168.0.103 ceph-stor03
+192.168.0.104 ceph-stor04
+192.168.0.105 ceph-stor05
+```
+
+
+### Configure password-less root ssh login from the deployment host to all other hosts. Methods will vary, but typically involve generating a ssh-keypair on the deployment host and using ssh-copyid to distribute the public key to each other node.
+
+
+### Install "git",  and other dependencies using the distribution repositories:
+Install git
+```bash
+apt-get -y install git software-properties-common python-software-properties
+```
+
+
+### Clone the rpc-ceph repository.  Standard practice is to put this under the /opt directory
+To see the versions of `ansible` and `ceph-ansible` supported in this version view the ~~README.md~ file.
+```bash
+cd /opt
+git clone  https://github.com/rcbops/rpc-ceph.git ; cd rpc-ceph
+```
+
+
+### Setup the virtual environment, create wrappers clone necessary repos and install ansible
+Run bootstrap-ansible.sh
+```bash
+./scripts/bootstrap-ansible.sh
+```
+
+
+### Create an ansible inventory file.  
+This file should be given a meaningful name for the deployment (ie: rs_clouda_dev_inventory, customera_prod_inventory, etc).  The inventory must contain sections for all "roles" which are to be deployed ("mon", "osds", "rgws", "mdss"). Each server may appear multiple times, if it is to receive multiple roles.  These name must match those listed in the /etc/hosts file.
+
+Example of an ansible inventory file
+```bash
+[all]
+ceph-stor01
+ceph-stor02
```

Comment:
I think if we're not going to do host_vars we need ansible_ssh_host

Reply:
Added these entries into the sample inventory file.    I'm really not quite sure about it.  

Fixed but we probably need to review this with the Deployment Team